config/nvim/init.vim

```diff
@@ -60,7 +68,11 @@ call plug#begin('~/.config/nvim/plugged')
     set showbreak=↪
     set autoindent " automatically set indent of new line
     set ttyfast " faster redrawing
-    set diffopt+=vertical,iwhite,internal,algorithm:patience,hiddenoff
+    if g:os == "Darwin"
+        set diffopt+=vertical,iwhite,internal,algorithm:patience,hiddenoff
+    elseif g:os == "Linux"
+        set diffopt=vertical,iwhite
```

Comment:
Why would this value be based on OS?

Reply:
What I would like to have done is add the options available in the nvim installation but I couldn't find the way, so I added the options available in the package manager installation of nvim in ubuntu testing for the os. Definitely not the best solution but I couldn't find a better one.